front_end/helper.py

```diff
@@ -241,3 +241,18 @@ def show_hidden(request_handler):
     if "show_hidden" not in request_handler.request.query_arguments:
         return False
     return request_handler.request.query_arguments["show_hidden"][0].decode().lower() == "true"
+
```

Comment:
I don't think these functions are being used. But even if they are, we could get by without them using something like this:

course_title = get_course_basics(course)["title"]

Reply:
I removed those functions.